js/flightlog_fields_presenter.js

```diff
@@ -732,6 +739,20 @@ function FlightLogFieldPresenter() {
                     return (value / 10).toFixed(1) + 'deg';
                 case 'AC_CORRECTION':
                     return (value / 10).toFixed(1) + 'deg/s';
+                case 'GPS_RESCUE_THROTTLE_PID':
+                        return value.toFixed(0);
+                case 'RTH':
+                    switch(fieldName) {
+                        case 'debug[0]':
+                            return value.toFixed(0);
+                        case 'debug[1]':
+                            return (value / 100).toFixed(1);
+                        case 'debug[2]':
+                            return value.toFixed(0);
+                        case 'debug[3]':
+                            return value.toFixed(0);
```

Comment:
If you have the with equal code, you can group them in a `default` group.

Reply:
Ok, now it looks better